lib/ansible/plugins/connection/docker.py

```diff
@@ -209,12 +211,54 @@ def exec_command(self, cmd, in_data=None, sudoable=False):
 
         local_cmd = self._build_exec_cmd([self._play_context.executable, '-c', cmd])
 
-        display.vvv("EXEC %s" % (local_cmd,), host=self._play_context.remote_addr)
-        local_cmd = [to_bytes(i, errors='surrogate_or_strict') for i in local_cmd]
-        p = subprocess.Popen(local_cmd, shell=False, stdin=subprocess.PIPE,
-                             stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        display.vvv(u"EXEC {0}".format(to_text(local_cmd)), host=self._play_context.remote_addr)
```

Comment:
Please keep this as it was before. Also, please get rid of the next line.

Reply:
Should be all set.